### ISSUE TITLE
Instagram responded with HTTP error "429 - Too Many Requests". Please do not run multiple instances of Instaloader in parallel or within short sequence. Also, do not use any Instagram App while Instaloader is running. The request will be retried in 666 seconds, at 22:26.

### ISSUE THREAD

**Issue by monajalal** [instaloader/instaloader#895]
```
(base) mona@mona:~/research/insta-scrape$ python instaloader_tester.py 

HTTP redirect from https://www.instagram.com/explore/tags/book/ to https://www.instagram.com/accounts/login/
JSON Query to explore/tags/book/: 429 Too Many Requests: redirected to login [retrying; skip with ^C]
Requests within last 10/11/20/22/30/60 minutes grouped by type:
 *                            other:    1    1    1    1    1    1
Instagram responded with HTTP error "429 - Too Many Requests". Please
do not run multiple instances of Instaloader in parallel or within
short sequence. Also, do not use any Instagram App while Instaloader
is running.
The request will be retried in 666 seconds, at 22:26.
```

for the code:

```
import instaloader

loader= instaloader.Instaloader(download_comments=False,
                    download_videos=False,
                    download_geotags=False,
                    save_metadata=False) 
loader.download_hashtag('book', max_count=100, fast_update=False, profile_pic=False)
```

```
$ instaloader --version
4.5.4
```

How can I fix this?

**Comment by aandergr** [comment]
This is fixed with Version 4.5.5.

**Comment by arungarg14** [comment]
I am using 4.8.2, 

but that issue is still showing
